src/main/app/src/app/admin/model/formulier-definitie-verwijzing.ts

```diff
@@ -5,5 +5,4 @@
 
 export class FormulierDefinitieVerwijzing {
```

Comment:
Is deze classe nog wel nodig? Kan deze niet vervangen worden door een string en naam "formulierDefinitieId"